### ISSUE TITLE
TOD-BERT: Pre-trained Natural Language Understanding for Task-Oriented Dialogues

### ISSUE THREAD

**Issue by toriving** [toriving/Plz_Read_The_Paper#156]
https://arxiv.org/pdf/2004.06871.pdf

![image](https://user-images.githubusercontent.com/42434734/121983178-db865080-cdcb-11eb-80f6-0ea55ed632cf.png)

![image](https://user-images.githubusercontent.com/42434734/121983186-dfb26e00-cdcb-11eb-9673-96f93d9efabc.png)

**Comment by toriving** [comment]
https://github.com/toriving/Plz_Read_The_Paper/blob/master/PAPERS/TOD-BERT-%20Pre-trained%20Natural%20Language%20Understanding%20for%20Task-Oriented%20Dialogue.pdf

**Comment by toriving** [comment]
- 기존 pre-trained BERT에 TOD task dataset을 모아 한번더 학습 시킨 모델
- 기존 버트와 유사하게 MLM 과 RSL (Response Selection Loss) 를 사용하여 학습하여 TOD task에 잘 맞도록 함
- TOD Task에서 Sota를 달성하며, Few-shot setting에서의 좋은 효과도 보여줌
- 이 논문은 "Don't Stop Pretraining: Adapt Language Models to Domains and Tasks" 논문에서의 Task-Adaptive Pretraininig (TAPT)의 케이스. (해당 논문에서 Domain-Adaptive Pretraining (DAPT)도 이야기함)
